infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/main.tf

```diff
@@ -125,7 +125,7 @@ module "ghproxy_nodepool" {
   initial_count   = 1
   min_count       = 1
   max_count       = 1
-  machine_type    = "n1-standard-8"
+  machine_type    = "e2-standard-4"
```

Comment:
What's the reasoning behind this change? I suspect this will constrain IOPS which may not be good for a caching service.

Reply:
OK, I see https://github.com/kubernetes/k8s.io/pull/921#issuecomment-636195722

Thinking about this a bit more, I suppose we can try not blindly replicating the setup used by prow.k8s.io for this setup.  

The caching here is going to be for the benefit of jobs, so I'm not sure I expect super high performance or super high availability.  How about we consider dropping this separate pool entirely, and just deploying as a resource on the existing node pool?

Reply:
I'm in favor of less dedicated pools, when possible.  It should be last-resort in most cases.

Reply:
PR updated with nodepool for ghproxy removed.